### PR TITLE
Update COC in upstream

### PR DESCRIPTION
**Pull request by amelia2802**
Resolved issue #66

### CODE REVIEW

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,76 @@
+# Contributor Covenant Code of Conduct
+
+Agro Millers is committed to providing a safe, respectful and inclusive environment for all its contributors, collaborators, and users. We expect all members of our community to abide by the following code of conduct:
```
Comment (AmanNegi) [inline CODE_OF_CONDUCT.md:3]: It seems like you misspelled the name of the application. Please change _AgroMillers_ to **_AgroMillets_** everywhere.

**Review by AmanNegi — CHANGES_REQUESTED** [review]
Please correct the name as mentioned. Rest all looks good to me 😊

File: CODE_OF_CONDUCT.md (hunk 1 above)
Comment (amelia2802) [reply CODE_OF_CONDUCT.md]: My bad. I haven't proofread the file! I have updated the file. You can check it now.

**Review by AmanNegi — APPROVED** [review]
LGTM!

**Comment by AmanNegi** [comment]
Good work! Thanks a lot for contributing, I will merge your PR right away 😊✨

**Comment by amelia2802** [comment]
Thank you for giving me the opportunity to contribute :)

**Pull request merged**